src/features/randomProfile/randomProfile.js

```diff
@@ -1,6 +1,5 @@
 import $ from "jquery";
 import { getRandomProfile } from "../../core/common";
-import { getPerson } from "wikitree-js";
```

Comment:
Is this a conflict with PR https://github.com/wikitree/wikitree-browser-extension/pull/119 ?

Reply:
No, I don't think so.  I found I needed that in the common.js file for getRandomProfile, which I moved to the common.js file to give the My Menu feature access to it.  I hope that makes sense, and I hope the getPerson call is in common.js now (I don't have time to check).  I'll be back later.